### PR TITLE
Remove "YA_FSM"

### PR DESCRIPTION
**Pull request by per1234**
Companion to https://github.com/arduino/library-registry/pull/4147

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
This is on hold pending an explanation of why the library maintainer closed the pull request on the `main` branch:

https://github.com/arduino/library-registry/pull/4147#pullrequestreview-1919976239

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/03/07 16:47:43 Loaded 6727 libraries from DB
Removing YA_FSM
Success!
```

**Pull request merged**
